models/Session.js

```diff
@@ -0,0 +1,41 @@
+var mongoose = require('mongoose');
+
+let SessionSchema = new mongoose.Schema({
```

Comment:
Que representa un session? A lo mejor poner un small comment arriba de cada schema para saber que representa no estaria mal

Reply:
Si, tiene sentido. Un session es basicamente la sesion de un usuario. So they log in and it creates a session, until it either expires or they log out